ci/scripts/README.md

```diff
@@ -0,0 +1,36 @@
+# Scripts
```

Comment:
This should be consolidated into `RELEASE.md` in `.github/`. I don't see why this information needs to be in more than one place.

Reply:
consolidated into `.github/RELEASE.md`